wordpress-mobile/WordPress-iOS
# Numerous color palette tweaks

Refs #11683

This changes numerous colors throughout the app to improve the look with the new color palette. I can provide a PDF of changes for review, if you'd like.

<img width="440" alt="image" src="https://user-images.githubusercontent.com/517257/60834430-88360d00-a175-11e9-81a8-5ee50d7af3d5.png">
<img width="425" alt="image" src="https://user-images.githubusercontent.com/517257/60834439-8c622a80-a175-11e9-9c47-96ff0445fa75.png">
<img width="411" alt="image" src="https://user-images.githubusercontent.com/517257/60834456-94ba6580-a175-11e9-8c55-d327c0464630.png">


To test:
- Build app and look through all the top-level screens. Particularly:
  - all links should be the new blue
  - all table views should have the new gray background
  - more text should be very dark, and a more true neutral, no dark-blue text, etc.

Update release notes:
